casepro/backend/junebug.py

```diff
@@ -463,3 +468,55 @@ def received_junebug_message(request):
         created_on=datetime.now(), has_labels=True)
 
     return JsonResponse(msg.as_json())
+
+
+@csrf_exempt
+def receive_identity_store_optout(request):
+    '''Handles optout notifications from the Identity Store.'''
+    if request.method != 'POST':
+        return JsonResponse({'reason': 'Method not allowed.'}, status=405)
+
+    try:
+        data = json.loads(request.body)
+    except ValueError as e:
+        return JsonResponse(
+            {'reason': 'JSON decode error', 'details': e.message}, status=400)
+
+    try:
+        identity_id = data['identity']
+        optout_type = data['optout_type']
+    except KeyError as e:
+        return JsonResponse(
+            {'reason': 'Both "identity" and "optout_type" must be specified.'},
+            status=400)
+
+    syncer = IdentityStoreContactSyncer()
+
+    # The identity store currently doesn't specify the response format or do
+    # anything with the response.
+
+    # TODO: Support existance and usage of multiple orgs
+    org = Org.objects.first()
```

Comment:
You can get the org from `request.org`.

If you use the url `org1subdomain.casepro.org` and `or2subdomain.casepro.org` then `request.org` gets populated with `org1` or `org2`, depending on the URL. If they don't use the subdomain, the get a redirect to select the org anyway, so you don't need to worry about that case.